docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md

```diff
@@ -0,0 +1,79 @@
+# Use GitHub as main hub for the Opinionated Digital Center
+
+* Status: proposed
+* Date: 2020-04-02
+
+## Context and Problem Statement
+
+We want to share the Opinionated Digital Center's (ODC) opinions, code, templates,
+pipelines and tools with the community. We want to make it as easy as possible to access
+and to contribute to, either by exchanging ideas, submitting issues and Pull/Merge
+Requests. Hence, we need to choose a hub.
+
+## Decision Drivers
+
+* Our main audience target are environments and companies using self-hosted or
+  cloud versions of GitLab or GitHub for hosting the code, and GitLab CI for pipelines.
```

Comment:
Also Jenkins for pipelines. Even if we explicitely tell why we will not do anything special for Jenkins

Reply:
I added stuff, let me know what you think about it.